Libraries/LibCrypto/Cipher/Mode/CTR.h

```diff
@@ -92,7 +93,7 @@ class CTR : public Mode<T> {
 public:
     constexpr static size_t IVSizeInBits = 128;
 
-    virtual ~CTR() { }
+    virtual ~CTR() {}
```

Comment:
This looks wrong, are you using clang-format 10?

Reply:
I did indeed! Thanks, this also removes all other annoying false-positives.